AVL_Tree/Cpp/avl_tree.cpp

```diff
@@ -0,0 +1,243 @@
+#include < iostream > #include < cstdio > #include < sstream > #include < algorithm > #define pow2(n)(1 << (n))
+using namespace std;
+
+/*
+ * Node Declaration
+ */
+struct avl_node {
+  int data;
+  struct avl_node * left;
+  struct avl_node * right;
+}* root;
+
```

Comment:
public: should be indented +1 space inside class avlTree  [whitespace/indent] [3]

*Origin: CPPLintBear, Section: `all.cpp`.*